### ISSUE TITLE
FTP password is undefined

### ISSUE THREAD

**Issue by refactoringg** [is-a-dev/register.is-a.dev#123]
**Expected:**
the password for ftp instead of "undefined"

**Got result**
![image](https://github.com/is-a-dev/register.is-a.dev/assets/113823517/0fce3f7e-39f4-4b31-a5d3-07c8894f1cb1)

**Comment by andrewstech** [comment]
Thats odd. I can't see your domain directory on the server did you click the create webserver button?

**Comment by refactoringg** [comment]
Yep, did do it.

**Comment by andrewstech** [comment]
> Yep, did do it.

Ok try visiting https://files.hosts.is-a.dev/  Its our online file editor (Its a work in progress) Login with github then select your domain. There will be a file called config.json click that and you should see your FTP password.

**Comment by refactoringg** [comment]
Alright thanks, will check it today.

**Comment by refactoringg** [comment]
![image](https://github.com/is-a-dev/register.is-a.dev/assets/113823517/2a6727ed-7bf7-4975-9cc1-e6779d8c7cda)

Hey mate, I'm getting this error when I'm trying to log into the FTP server.

**Comment by andrewstech** [comment]
> ![image](https://user-images.githubusercontent.com/113823517/260228951-2a6727ed-7bf7-4975-9cc1-e6779d8c7cda.png)
> 
> Hey mate, I'm getting this error when I'm trying to log into the FTP server.

Thats very odd. what FTP client have you tried using. Unfortunately we only introduced error tracking today so we don't have any logs.

I have just tried your account with the default password that was generated for you and it seems to be working.

<img width="960" alt="image" src="https://github.com/is-a-dev/register.is-a.dev/assets/45342431/a5da018f-dfde-4a72-9f86-493f563a8917">
 

Try again and if it doesn't work then i can regenerate your config
